nodejs/week1/src/backend/index.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const meals = require("./data/meals.json");
+const reservations = require("./data/reservations.json");
+
+const app = express();
+
+app.get("/meals", (req, res)=>{
+    res.json(meals);
+});
+
+app.get("/reservations", (req, res)=>{
+    res.json(reservations);
+});
+
+app.get("/cheap-meals", (req, res)=>{
+    cheapMeals = meals.filter((cheapmeal)=>{
+        return cheapmeal.price < 65;
+    });
+    res.json(cheapMeals);
+});
+
+app.get("/large-meals", (req, res)=>{
+    largeMeals = meals.filter((largermeal)=>{
+        return largermeal.maxNumberOfGuests>7;
+    });
+    res.json(largeMeals)
+});
+
+app.get("/meals", (req, res)=>{
+    const randommeal = meals[Math.floor(Math.random() * meals.length)];
```

Comment:
I would extract a variable:

const randomIndex = Math.floor(Math.random() * meals.length);

for added clarity